Article.js

```diff
@@ -0,0 +1,32 @@
+class Article {
+  constructor(title,content,writer) {
+    this.title = title;
+    this.content = content;
+    this.writer = writer;
+    this.likeCount = 0;
+    this.createdAt = new Date();
+    }
+  like() {
+    this.likeCount+=1
+    }
+  get title() {
+    return this._title;
+  }
+  set title(newTitle) {
+    if (typeof newTitle === 'string' && newTitle.trim() !== '') {
+      this._title = newTitle;
+    } else {
+      throw new Error('이름은 비어있을 수 없습니다.');
+    }
+  }
+
+  get content() {
+    return this._content; 
+  }
+
+  get writer() {
+    return this._writer;
+  }
+  }
+
+

```

Comment:
현정님이 의도하신 방향대로, 코드 리뷰를 정리하면 아래처럼 작성할 수 있어요. 변경된 부분은
- 캡슐화
- like()를 통해서만 likeCount 제어. `article.#likeCount = 10` 등의 코드는 SyntaxError를 뱉어내기 때문에 안전해요.

```suggestion
class Article {
  #title;
  #content;
  #writer;
  #likeCount = 0;

  constructor(title, content, writer) {
    this.title = title;
    this.content = content;
    this.writer = writer;
    this.createdAt = new Date();
  }

  like() {
    this.#likeCount += 1;
    return this.#likeCount;
  }

  get likeCount() {
    return this.#likeCount;
  }

  get title() {
    return this.#title;
  }

  set title(newTitle) {
    if (typeof newTitle === 'string' && newTitle.trim() !== '') {
      this.#title = newTitle;
    } else {
      throw new Error('제목은 빈 문자열일 수 없습니다.');
    }
  }

  get content() {
    return this.#content;
  }
  get writer() {
    return this.#writer;
  }
}
```